api/observability/v1/output_types.go

```diff
@@ -769,3 +774,50 @@ type Syslog struct {
 	// +kubebuilder:validation:Optional
 	MsgID string `json:"msgID,omitempty"`
 }
+
+type OtlpTuningSpec struct {
+	BaseOutputTuningSpec `json:",inline"`
+
+	// Compression causes data to be compressed before sending over the network.
+	// It is an error if the compression type is not supported by the output.
+	//
+	// +kubebuilder:validation:Enum:=gzip;none
+	// +kubebuilder:default:=gzip
+	Compression string `json:"compression,omitempty"`
+}
+
+// Otlp provided configuration for sending OTLP/HTTP with JSON-encoded Protobuf payload requests.
+// https://opentelemetry.io/docs/specs/otlp/#json-protobuf-encoding
+type Otlp struct {
+	// URL to send log records to.
+	//
+	// An absolute URL, with a valid http scheme. Should terminate with `/v1/logs`
+	//
+	// Basic TLS is enabled if the URL scheme requires it (for example 'https').
+	// The 'username@password' part of `url` is ignored.
+	//
+	// +kubebuilder:validation:Required
+	URL string `json:"url"`
+
+	// Authentication sets credentials for authenticating the requests.
+	//
+	// +kubebuilder:validation:Optional
+	Authentication *HTTPAuthentication `json:"authentication,omitempty"`
+
+	// Tuning specs tuning for the output
+	//
+	// +kubebuilder:validation:Optional
+	// +nullable
+	Tuning *OtlpTuningSpec `json:"tuning,omitempty"`
+
+	// Headers specify optional headers to be sent with the request
+	// default value is: {"Content-Type"="application/json"}
+	//
+	// +kubebuilder:validation:Optional
+	Headers map[string]string `json:"headers,omitempty"`
+
+	// Timeout specifies the Http request timeout in seconds. If not set, 10secs is used.
+	//
+	// +kubebuilder:validation:Optional
+	Timeout int `json:"timeout,omitempty"`
```

Comment:
This is not part of the enhancement API.  Do we have need to support this initially?

Reply:
Nope